Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/docker/DockerContainerNode.java

```diff
@@ -256,7 +256,7 @@ protected void onSubscriptionsChanged(NodeActionEvent e) throws AzureCmdExceptio
     }
 
     @Override
-    protected String getServiceName() {
+    protected String getServiceName(NodeActionEvent event) {
       return DOCKER;
```

Comment:
Some of Node uses static import while others specifying the class, maybe we could unify the code style later.